contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -185,6 +365,93 @@ NXmpes(NXobject):
           exists: recommended
           doc: |
             This is the momentum axis to be used for data plotting.
+      energy_referencing(NXcalibration):
+        exists: optional
+        doc:
+        - |
+          For energy referencing, the measured energies are corrected for the charging potential
+          (i.e., the electrical potential of the surface region of an insulating sample, caused by
+          irradiation) such that those energies correspond to a sample with no surface charge.
+          Usually, the energy axis is adjusted by shifting all energies uniformally until one
+          well-defined emission line peak (or the Fermi edge) is located at a known _correct_ energy.
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 12.74 ff.
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.74
+        applied(NX_BOOLEAN):
+          doc: |
+            Have the energy axes been adjusted (e.g., in cases where samples are not
+            conductive)?
+        level(NXelectron_level):
+          exists: recommended
+          doc: |
+            Electronic core or valence level that was used for the calibration.
+        reference_peak:
+          doc: |
+            Reference peak that was used for the calibration.
+            
+            For example: adventitious carbon | C-C | metallic Au | elemental Si | Fermi edge
```

Comment:
Isn't that redundant or partially overlapping?

Reply:
The idea was that you provide an electron level region (i.e., C 1s) and then you also need to specify which specific peak in that region you are using for energy correction (i.,e., the peak that arises due to C-C bonding).

Reply:
I guess for the Fermi edge. these two would be the same, you are right.